test/controllers/MessagesControllerSpec.scala

```diff
@@ -327,7 +332,9 @@ class MessagesControllerSpec extends SpecBase with ScalaCheckPropertyChecks with
         running(application) {
           lazy val request = FakeRequest(GET, routes.MessagesController.getMessages(departure.departureId).url)
           val result       = route(application, request).value
+          val body         = Json.obj()
 
+          contentAsJson(result) must not be body
```

Comment:
These lines can be removed as contentAsJson is being used later in the test case.